File: sprint8/By languages/JS/lesson2code1.js
function compare(first, second) {
    if (first.length !== second.length) {
        return false;
    }

    var length = first.length;
    for (var i = 0; i < length; ++i) {
        if (first[i] !== second[i]) {
            return false;
        }
    }

    return true;
}

File: sprint8/By languages/kotlin/lesson2code1.kt
fun compare(first: String, second: String): Boolean {
    if (first.length != second.length) {
        return false
    }

    val length = first.length
    for (i in 0 until length) {
        if (first[i] != second[i]) {
            return false
        }
    }

    return true
}
